shanoir-uploader/src/main/java/org/shanoir/uploader/action/UploadFromFolderActionListener.java

```diff
@@ -0,0 +1,96 @@
+package org.shanoir.uploader.action;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.log4j.Logger;
+import org.shanoir.uploader.gui.ImportFromFolderWindow;
+import org.shanoir.uploader.model.ExaminationImport;
+import org.shanoir.uploader.model.FolderImport;
+import org.shanoir.uploader.model.rest.IdList;
+import org.shanoir.uploader.model.rest.Study;
+import org.shanoir.uploader.model.rest.StudyCard;
+import org.shanoir.uploader.service.rest.ShanoirUploaderServiceClient;
+import org.shanoir.uploader.utils.FileUtil;
+
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.ResourceBundle;
+
+public class UploadFromFolderActionListener implements ActionListener {
+
+    JFileChooser fileChooser;
+    ImportFromFolderWindow importFromFolderWindow;
+    private ResourceBundle resourceBundle;
+
+    private static Logger logger = Logger.getLogger(UploadFromCsvActionListener.class);
+
+    public UploadFromFolderActionListener(ImportFromFolderWindow importFromFolderWindow, ResourceBundle resourceBundle) {
+        this.importFromFolderWindow = importFromFolderWindow;
+        this.fileChooser = new JFileChooser("/home/jcome/Documents/Shanoir/Data");
```

Comment:
Hi @jcomedouteau, thank you for this great PR! Absolute path to your folder, I will change this on develop.